poh-bundle-linux-x32/oh.sh

```diff
@@ -144,24 +142,34 @@ SCRIPT_NAME=$(basename "$0")
 
 function script_usage {
 	# show help
-	echo ""
-	echo " Portable Open Hospital Client - OH"
-	echo ""
-	echo " Usage: $SCRIPT_NAME [-option]"
-	echo ""
-	echo "   -l    en|fr|it|es|pt   --> set language [default set to en]"
-	echo ""
-	echo "   -s    save OH database"
-	echo "   -r    restore OH database"
-	echo "   -c    clean POH installation"
-	echo "   -d    start POH in debug mode"
-	echo "   -C    start Open Hospital - Client / Server mode"
-	echo "   -t    test database connection (Client mode only)"
-	echo "   -v    show POH version information"
-	echo "   -D    start POH in Demo mode"
-	echo "   -G    Setup GSM"
-	echo "   -h    show this help"
-	echo ""
+
+
+        # show menu
+        # Clear-Host # clear console
+        echo " ---------------------------------------------------------"
+        echo "|                                                         |"
+        echo "|                   Open Hospital | OH                    |"
+        echo "|                                                         |"
+        echo " ---------------------------------------------------------"
+        echo " lang $OH_LANGUAGE | arch $ARCH"
+        echo ""
+        echo " Usage: $SCRIPT_NAME [ -lang en|fr|it|es|pt ] "
+#        echo "               [ -distro PORTABLE|CLIENT ]"
+#        echo "               [ -debug INFO|DEBUG ] "
+        echo ""
+        echo "   -s    save OH database"
+        echo "   -r    restore OH database"
+        echo "   -l    set language: en|fr|it|es|pt"
+        echo "   -C    start OH - CLIENT mode (Client / Server configuration)"
+        echo "   -t    test database connection (CLIENT mode only)"
+        echo "   -D    start OH in DEMO mode"
+        echo "   -d    start OH in DEBUG mode"
+        echo "   -c    clean OH installation"
+        echo "   -G    setup GSM"
+        echo "   -v    show OH software version and configuration"
+        echo "   -h    show this help"
+        echo "   -q    quit"
```

Comment:
Question, should we put in alphabetical order like in man?

Reply:
This is in alphabetical order (linux version):
 ---------------------------------------------------------
|                                                         |
|                   Open Hospital | OH                    |
|                                                         |
 ---------------------------------------------------------
 lang en | arch x86_64

 Usage: oh.sh.1 [ -lang en|fr|it|es|pt ] 

   -c    clean OH installation
   -C    start OH in CLIENT mode (Client / Server configuration)
   -d    start OH in DEBUG mode
   -D    start OH in DEMO mode
   -G    setup GSM
   -h    show this help
   -l    set language: en|fr|it|es|pt
   -r    restore OH database
   -s    save OH database
   -t    test database connection (CLIENT mode only)
   -v    show OH software version and configuration

What do you think ? Some options might be confused...

Reply:
it is very clear to me

Reply:
seen also the new option -X for the clean, ok for me!